products/eks.md

```diff
@@ -23,48 +23,54 @@ auto:
 -   custom: true
 
 # EOL dates can be found on https://docs.aws.amazon.com/eks/latest/userguide/kubernetes-versions.html#kubernetes-release-calendar
+# Latest release derived from tags in eks-distro Github repo https://github.com/aws/eks-distro
 releases:
+-   releaseCycle: "1.28"
```

Comment:
Are you sure about this one ? 1.28 is not listed on https://docs.aws.amazon.com/eks/latest/userguide/kubernetes-versions.html.

Reply:
I used the information from the release calendar on this page.  It's okay with me if it's dropped, at least for now, but do want you to see that the information came from AWS at least.

https://docs.aws.amazon.com/eks/latest/userguide/kubernetes-versions.html#kubernetes-release-calendar